wandb/sdk/internal/sender.py

```diff
@@ -263,8 +259,9 @@ def __init__(
         self._project = None
 
         # keep track of config from key/val updates
-        self._consolidated_config: DictNoValues = cast(DictNoValues, dict())
-        self._start_time: float = 0
+        self._consolidated_config = sender_config.ConfigState()
+
+        self._start_time: int = 0
```

Comment:
wait should start time be int?

Reply:
On line 900, I updated the calculation for it to use flooring division (`//`), since it feels a little odd to use a float for milliseconds. Technically a float would work, since I think Python floats are double-precision and the current Unix milliseconds is around 2^40. I don't expect this to cause issues when sending the dict to the backend if we do something like JSON-encode it, but maybe I'm wrong.